experiments/integrated/ozark/soil_retention_curve.jl

```diff
@@ -16,12 +16,13 @@ swrc_data = readdlm(data, ','; skipstart = 1)
 vwc = swrc_data[4:end, 1] ./ 100 # to convert from percent to a decimal
 ψ = swrc_data[4:end, 2] .* 1e6 ./ (1e3 * 9.8) # to convert from MPa to m
 
+FT = Float64
 # Assume value for this:
 θ_r = 0.04
 α = 10.0 .^ (log10(5e-3):0.1:1)
 n = Array(1.1:0.05:3.0)
 porosity = [0.45, 0.5, 0.55]
-L2 = zeros(Float64, (length(α), length(n)))
+L2 = zeros(FT, (length(α), length(n)))
```

Comment:
I dont think we need these changes really, this is a one off file for fitting parameters but isnt ever used in the simulation.

Reply:
okay, I'll remove them